modules/hivemq-edge-module-http/build.gradle.kts

```diff
@@ -36,40 +34,19 @@ repositories {
 }
 
 dependencies {
-    compileOnly("com.hivemq:hivemq-edge-adapter-sdk:${property("hivemq-edge-adapter-sdk.version")}")
-    compileOnly("commons-io:commons-io:${property("commons-io.version")}")
-
-    implementation("org.slf4j:slf4j-api:${property("slf4j.version")}")
-    implementation("com.fasterxml.jackson.core:jackson-databind:${property("jackson.version")}")
-    runtimeOnly("com.google.guava:guava:${property("guava.version")}") {
```

Comment:
guava has been removed, I guess it was not needed

Reply:
yeah